### DISCUSSION TITLE
Audio on load 3D model

### DISCUSSION THREAD

**Discussion by mihirm3hub** [google/model-viewer#2721] [Q&A]
How to play audio after loading the 3D model, I have checked the synced audio discussion and it was of great help. The model I am trying to work on is bigger in size and takes time to load so is it possible to play audio after the model is loaded?

**Comment by elalish** [comment]
I believe you're looking for our `load` event: https://modelviewer.dev/docs/index.html#entrydocs-loading-events-load

**Reply by mihirm3hub** [reply]
@elalish I think so, but how can I implement it ?? to play sound

**Reply by elalish** [reply]
If you're not familiar with using event listeners to trigger JS functions, I'd recommend reading up on that. It's pretty easy once you've got the basics.
